client/src/layout/MentionList.js

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from "react";
+import {
+  Switch,
+  List,
+  ListItem,
+  Divider,
+  Avatar,
+  ListItemText,
+  ListItemAvatar,
+  Typography,
+  ListItemSecondaryAction,
+} from "@material-ui/core";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
+import reddit from "../utils/images/reddit-logo.png";
+import twitter from "../utils/images/twitter-logo.png";
+import facebook from "../utils/images/facebook-logo.png";
+import amazon from "../utils/images/amazon-logo.png";
+import forbes from "../utils/images/forbes-logo.png";
+import shopify from "../utils/images/shopify-logo.png";
+import business_insider from "../utils/images/BI-logo.png";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+  },
+  listItem: {
+    height: "5em",
+    marginLeft: theme.spacing(2),
+  },
+}));
+
+const CustomizedSwitch = withStyles((theme) => ({
+  root: {
+    width: 50,
+    height: 25,
+    padding: 0,
+    marginRight: theme.spacing(2),
+  },
+  switchBase: {
+    padding: 3,
+    "&$checked": {
+      transform: "translateX(23px)",
+      color: theme.palette.common.white,
+      "& + $track": {
+        backgroundColor: theme.palette.primary.main,
+        opacity: 1,
+      },
+    },
+  },
+  thumb: {
+    width: 20,
+    height: 20,
+  },
+  track: {
+    borderRadius: 25 / 2,
+    border: "none",
+    backgroundColor: "#bdc7d4",
+    opacity: 1,
+  },
+  checked: {},
```

Comment:
These can also be put in a/the same makeStyles, but you also might consider putting the CustomizedSwitch in its own file

Reply:
Yeah, CustomizedSwitch has been separated.